Added brief description for queries system stop|start merges (#7568)
ClickHouse/ClickHouse: docs/en/query_language/system.md

- [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends)
- [FLUSH DISTRIBUTED](#query_language-system-flush-distributed)
- [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends)

> - [STOP|START MERGES](#query_language-system-stop-start-merges)

Reviewer: we'd better use separate descriptions for start and stop to make it consistent with "distributed sends" few lines above

SYSTEM START DISTRIBUTED SENDS [db.]<distributed_table_name>
```

### STOP|START MERGES {#query_language-system-stop-start-merges}

Provides possibility to stop and start background merges for all tables in the MergeTree family:

```sql

> SYSTEM STOP|START MERGES

Reviewer: we'd better use one syntax description that shows that table name is optional, then say that if it's ommitted the command will affect all tables with MergeTree engine

```

Optionally we can define specific table:  

```sql
SYSTEM STOP|START MERGES [db.]<merge_tree_family_table_name>
```

> NOTE: DETACH / ATTACH table will start background merges for the table even in case we stopped merges for all *MergeTree tables before).

Reviewer: ```suggestion
!!! note "Note":
    `DETACH / ATTACH` table will start background merges for the table even in case when merges have been stopped for all *MergeTree tables before.
```
